bin/pycbc_live

```diff
@@ -237,9 +237,9 @@ class LiveEventManager(object):
                 gid = None
                 event.save(fname)
 
-            if self.run_snr_optimization and gid is not None:
```

Comment:
Would this change mean that now the SNR optimization is run more? Previously it would not run if `self.ifar_upload_threshold < ifar`.

Reply:
Yes, we probably want
```python
if self.run_snr_optimization and ifar < self.ifar_upload_threshold:
```